.github/workflows/compile-fw.yml

```
@@ -4,11 +4,14 @@ on:
   pull_request:
     paths:
       - ".github/workflows/compile-fw.yml"
+      - "./l3xz-fw_leg-controller.ino"
```

```suggestion
      - "l3xz-fw_leg-controller.ino"
```
Relative paths can't be used in path filters.